test/reference-silgen_out.swift

```diff
@@ -0,0 +1,182 @@
+sil_stage canonical
```

Comment:
Not sure you wanted to submit this file to the upstream.

Reply:
Ah thank you! This was a mistake.